### PR TITLE
Properly reverse the calculation for fs overhead.

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Bugfix: Fix fsoverhead calculation
```

### CODE REVIEW

File: pkg/importer/data-processor.go [hunk 1]
```diff
@@ -372,11 +377,11 @@ func (dp *DataProcessor) getUsableSpace() int64 {
 
 // GetUsableSpace calculates space to use taking file system overhead into account
 func GetUsableSpace(filesystemOverhead float64, availableSpace int64) int64 {
-	spaceWithOverhead := int64((1 - filesystemOverhead) * float64(availableSpace))
+	// +1 always rounds up.
+	spaceWithOverhead := int64((1 - filesystemOverhead) * float64(availableSpace+1))
```
Comment (aglitke) [inline pkg/importer/data-processor.go:381]: What if it's already 123234.0?

**Review by aglitke — COMMENTED** [review]
Looks good despite the one nit

File: pkg/importer/data-processor.go (hunk 1 above)
Comment (awels) [reply pkg/importer/data-processor.go:381]: IIRC the easiest way to do this was to just add one. but I can do a int64(math.Ceil(calculation)) instead, not sure if it is any better. I have the cases where it is even covered in my unit testing.

**Comment by awels** [comment]
One failure is because we have an error in the storage.spec calculation that #1923 should be addressing https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/1924/pull-containerized-data-importer-e2e-k8s-1.20-nfs/1433864320474681344/artifacts/pods/3_cdi-e2e-tests-importer-hnvpz_importer-too-large-import-importer.log

**Comment by mhenriks** [comment]
superseded by https://github.com/kubevirt/containerized-data-importer/pull/1926

**Pull request closed without merging**